modules/installation-user-infra-machines-advanced-customizing-live-iso-network-config.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_network_keyfile_{context}"]
+= Modifying a live install image with customized network settings
+You can embed a NetworkManager keyfile into the live ISO image and pass it through to the installed system with the `--network-keyfile` flag.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+. Create a connection profile for the bonded interface. For example, create the `/etc/NetworkManager/system-connections/example.bond0` file with the following content:
```

Comment:
I think usually the files are all named `$id.nmconnection`. So e.g. here:

```suggestion
. Create a connection profile for the bonded interface. For example, create the `/etc/NetworkManager/system-connections/bond0.nmconnection` file with the following content:
```

And similarly below for the bond members (see e.g. the filenames in https://docs.fedoraproject.org/en-US/fedora-coreos/sysconfig-network-configuration/#_butane_config_2).